examples/simple.rs

```diff
@@ -1,40 +1,65 @@
-
 extern crate rspec;
-use rspec::context::describe;
+
 use std::io;
+use std::sync::{Arc, Mutex};
+use std::collections::BTreeSet;
 
-pub fn main() {
-    let stdout = &mut io::stdout();
-    let mut formatter = rspec::formatter::Simple::new(stdout);
-    let mut runner = describe("rspec is a classic BDD testing", |ctx| {
+use rspec::prelude::*;
 
-        ctx.it("can define tests", || true);
+pub fn main() {
```

Comment:
also: is this file concise enough ? Straight to the point ? Maybe the purpose f this example is not clear enough and needs to be split to show something precise ?